modules/debug_cmd/debug_cmd.c

```diff
@@ -56,7 +60,7 @@ static int print_system_info(struct re_printf *pf, void *arg)
 
 #ifdef USE_OPENSSL
 	err |= re_hprintf(pf, " OpenSSL:  %s\n",
-			  SSLeay_version(SSLEAY_VERSION));
+			  OpenSSL_version(OPENSSL_VERSION));
```

Comment:
is it not better to check for the version that the function was added ?

```c
#if VERSION >= X
OpenSSL_version(OPENSSL_VERSION);
#else
SSLeay_version(SSLEAY_VERSION);
#endif
```

Reply:
That's done above.